crates/fiber-lib/src/store/store_impl/native.rs

```diff
@@ -100,9 +114,73 @@ impl Store {
     }
 }
 
+pub struct BatchWatcher {
+    inner: Arc<dyn StoreChangeWatcher>,
+    pending_changes: Vec<StoreChange>,
```

Comment:
when remove changes from `pending_changes`.

Reply:
Batch is destroyed after calling `commit`.